docs/admin/runai-setup/cluster-setup/cluster-install.md

```diff
@@ -42,47 +42,73 @@ Example output:
       status: "True"
       reason: reconciled
       message: Reconciliation completed successfully
-  optional:  # (2)
-    knative:    # (3)  
+  optional:  # (3)
+    knative:    # (4)  
       components:
         hpa:
           available: true
         knative:
           available: true
         kourier:
           available: true
-    mpi:        # (4) 
+    mpi:        # (5) 
       available: true
 ```
 
-1. Verifies that all mandatory dependencies are met: NVIDIA GPU Operator, Prometheus and NGINX controller. 
-2. Checks whether optional product dependencies have been met.
-3. See [Inference prerequisites](cluster-prerequisites.md#inference).
-4. See [distributed training prerequisites](cluster-prerequisites.md#distributed-training).
+1. Verifies that all mandatory dependencies are met: NVIDIA GPU Operator, Prometheus and NGINX controller.
+2. Verifies that all of Run:ai managed resources have been successfully deployed.
+3. Checks whether optional product dependencies have been met.
+4. See [Inference prerequisites](cluster-prerequisites.md#inference).
+5. See [distributed training prerequisites](cluster-prerequisites.md#distributed-training).
 
 For a more extensive verification of cluster health, see [Determining the health of a cluster](../../troubleshooting/cluster-health-check.md).
 
+### Troubleshooting
+#### Dependencies are not fulfilled
+1. Make sure to install the missing dependencies.
+2. If dependencies are installed, make sure that the CRDs of said dependency are installed, and that the version is supported
+3. Make sure there are no necessary adjustments for specific flavors as noted here https://docs.run.ai/v2.15/admin/runai-setup/cluster-setup/cluster-prerequisites
+
+#### Resources not deployed / System Unavailable / Reconciliation Failed
+1. Run Preinstall diagnostic script https://docs.run.ai/v2.15/admin/runai-setup/cluster-setup/cluster-prerequisites/#pre-install-script and check for issues
+2. Run
+```
+   kubectl get pods -n runai
+   kubectl get pods -n monitoring
+```
+and obtain logs from any failing pod `kubectl logs <pod_name>`
+
+#### Common Issues
+1. Run:ai was previously installed in the cluster and was deleted unsuccessfully, resulting in remaining CRDs
+    2. Diagnosis: This can be detected by running `kubectl get crds` in the relevant namespaces (or adding `-A` and checking for Run:ai CRDs)
+    3. Solution: Force delete said CRDs and reinstall
+3. One or more of the pods have issues around valid certificates
+    4. Diagnosis: The logs contains a message similar to the following `failed to verify certificate: x509: certificate signed by unknown authority`
+    4. Solution:
+        5. This is usually due to an expired or invalid certificate in the cluster, and if so - renew the certificate
+        6. If Certificate is valid, but is Signed by a local CA, make sure to have followed the steps here https://docs.run.ai/v2.15/admin/runai-setup/config/org-cert/
+
```

Comment:
@roir I am going to nee to merge this PR to check some of the markdown. I don't have a good way of previewing all the indentations to ensure that it renders correctly.
```suggestion
#### Common Issues

1. Run:ai was previously installed in the cluster and was deleted unsuccessfully, resulting in remaining CRDs.
    1. Diagnosis: This can be detected by running `kubectl get crds` in the relevant namespaces (or adding `-A` and checking for Run:ai CRDs).
    2. Solution: Force delete the listed CRDs and reinstall.
2. One or more of the pods have issues around valid certificates.
    1. Diagnosis: The logs contains a message similar to the following `failed to verify certificate: x509: certificate signed by unknown authority`.
    2. Solution:
        1. This is usually due to an expired or invalid certificate in the cluster, and if so, renew the certificate.
        2. If the certificate is valid, but is signed by a local CA, make sure you have followed the procedure for a [local certificate authority](../../config/org-cert.md).
```